code/controllers/subsystem/processing/fluids.dm

```diff
@@ -1,5 +1,5 @@
 PROCESSING_SUBSYSTEM_DEF(fluids)
 	name = "Fluids"
-	wait = 20
+	wait = 10
 	stat_tag = "FD" //its actually Fluid Ducts
-	flags = SS_NO_INIT | SS_TICKER
+	flags = SS_NO_INIT
```

Comment:
This isn't completely the same behavior wise, just so you know

Reply:
i know, i dont see why SSfluids warrants the ordering bump that SS_TICKER gives it though